routers/repo/issue.go

```diff
@@ -1542,6 +1549,12 @@ func ChangeCommentReaction(ctx *context.Context, form auth.ReactionForm) {
 
 	switch ctx.Params(":action") {
 	case "react":
+		if !util.IsStringInSlice(form.Content, setting.UI.Reactions) {
+			log.Error("ChangeIssueReaction: " + form.Content + " is no allowed reaction")
```

Comment:
Similarly to above. 

Just use ctx.Flash.Error and redirect back

Reply:
@zeripath how to redirect back?
this code get only executed when kliking on a reaction emoji

-> ctx.Flush.Error dont produce an erreror message